utils/spacewalk-common-channels.ini

```diff
@@ -2949,3 +2949,59 @@ gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni
 gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/CentOS8-Uyuni-Client-Tools/CentOS_8/
+
+
+[rockylinux8]
+archs    = x86_64
+name     = RockyLinux 8 (%(arch)s)
+gpgkey_url = https://dl.rockylinux.org/pub/rocky/RPM-GPG-KEY-rockyofficial
+gpgkey_id = 6D745A60
+gpgkey_fingerprint = 7051 C470 A929 F454 CEBE 37B7 15AF 5DAC 6D74 5A60
+repo_url = https://dl.rockylinux.org/pub/rocky/8/BaseOS/%(arch)s/os/
```

Comment:
I think, here and in the following sections, should really be used the mirrorlist, to get better performance out of the box.
Yes the Tier 0 Mirror `dl.rockylinux.org` is a CDN, but this has a limited capacity in some locations (and isn't even available in others)
So in this case it should be `https://mirrors.rockylinux.org/mirrorlist?repo=BaseOS-8&arch=%(arch)s`

Reply:
I will fix it. I requested @digdilem permission to write on his fork, to do the fixes for the PR, and maybe some enhacements.

Reply:
Fixed (pending testing).